pkg/controllers/secrets.go

```diff
@@ -178,7 +178,7 @@ func ReadTemplateFile(filePath string) string {
 
 func RenderSecretsTemplate(templateBody string, secretsMap map[string]string) string {
 	funcs := map[string]interface{}{
-		"tojson": func(value string) (string, error) {
+		"tojson": func(value interface{}) (string, error) {
```

Comment:
Could you add a `chore:` prefix to this commit too? ("Allow tojson to operate on any value")